docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md

```diff
@@ -0,0 +1,219 @@
+---
+description: Learn how to get started with Grafana Tempo using the Helm chart.
+menuTitle: Get started
+title: Get started with Grafana Tempo using the Helm chart
+weight: 20
+keywords:
+  - Helm chart
+  - Kubernetes
+  - Grafana Tempo
+---
+
+# Get started with Grafana Tempo using the Helm chart
+
+The Grafana Tempo Helm chart allows you to configure, install, and upgrade Grafana Tempo within a Kubernetes cluster. Using this procedure, you will:
+
+- Create a custom name-space within your Kubernetes cluster
+- Install Helm and the grafana helm-charts repository
+- Configure Google Cloud Platform storage for traces
+- Install Tempo using Helm
+
+To learn more about Helm, read the [Helm documentation](https://helm.sh/).
+
+## Before you begin
+
+These instructions are common across any flavor of Kubernetes. They also assume that you know how to install, configure, and operate a Kubernetes cluster.
+It also assumes that you have an understanding of what the `kubectl` command does.
+
+>**CAUTION**: This procedure is primarily aimed at local or development setups.
+
+### Hardware requirements
+
+- A single Kubernetes node with a minimum of 4 cores and 16GiB RAM
+
+### Software requirements
+
+- Kubernetes 1.20 or later (see [Kubernetes installation documentation](https://kubernetes.io/docs/setup/))
+- The `kubectl` command for your version of Kubernetes
+- Helm 3 or later (see [Helm installation documentation](https://helm.sh/docs/intro/install/))
+
+Verify that you have:
+- Access to the Kubernetes cluster
+- Persistent storage is enabled in the Kubernetes cluster, which has a default storage class setup. You can [change the default StorageClass](https://kubernetes.io/docs/tasks/administer-cluster/change-default-storage-class/).
+- Access to a Google Cloud Platform storage bucket (see [instructions](https://cloud.google.com/storage/docs/creating-buckets))
```

Comment:
This requirement breaks the local caution above.  Also, we probably want to say something to users who are in Azure or AWS, and not in GCP.

Reply:
I think we've addressed this comment by reorganizing the content for this procedure.